validate.go

```diff
@@ -208,6 +211,23 @@ func checkLinux(spec rspec.Spec) {
 			logrus.Fatalf("namespace %v is invalid.", spec.Linux.Namespaces[index])
 		} else if spec.Linux.Namespaces[index].Type == rspec.UTSNamespace {
 			utsExists = true
+		} else if spec.Linux.Namespaces[index].Type == rspec.IPCNamespace {
+			ipcExists = true
```

Comment:
These checks should probably skip namespaces where `path` is set (so they only pick up *new* namespaces).  If tweaking existing namespaces is fine, there's no need for the “does the config declare a namespace” checks.  And the current spec [explicitly forbids tweaking non-new namespaces][1]:

> Also, when a path is specified, a runtime MUST assume that the setup for that particular namespace has already been done and error out if the config specifies anything else related to that namespace.

Although [I'm happy to revisit that restriction][2].

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc1/config-linux.md#namespaces
[2]: https://github.com/opencontainers/runtime-spec/pull/158#issuecomment-138997548